### PR TITLE
Fix #93, fix #94. Add front end section and fix up multiplayer section

### LINKED ISSUE #93: Improve and move multiplayer analysis

### LINKED ISSUE #94: Add front end to the solution

### PR DESCRIPTION
**Pull request by Danar435**

### CODE REVIEW

File: src/problem-solution/frontend.tex [hunk 1]
```diff
@@ -0,0 +1,139 @@
+% textidote: ignore begin
+\section{Front end}\label{sec:frontend}
+% textidote: ignore end
+
+The following section is a showcase of the look and feel of the finished product.
+It'll go over every aspect of the website, from the landing page to the game interface, with explanations on how to
```
Comment (SebastianMygind) [inline src/problem-solution/frontend.tex]: Generally never use abbreviations in "serious" literature

File: src/problem-solution/frontend.tex [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+\end{figure}
+% textidote: ignore end
+
+Once the user is authenticated, the home page would lead to the lobby listing, rather than the landing page.
+The lobby listing is a list of all the available games that the user can join.
+As more users join the website, the lobby listing will be populated with more games, but during development, the lobby
+listing is empty, as seen in Figure~\ref{fig:lobby-empty}.
+
+Anyone can create a new game by clicking on the green ``Create New Match'' button in the top right corner of the listings.
+This will create add a new entry to the lobby, as seen in Figure~\ref{fig:lobby-game}.
```
Comment (SebastianMygind) [inline src/problem-solution/frontend.tex]: create or add not both

**Review by SebastianMygind — CHANGES_REQUESTED** [review]
looks good except two minor issues.
Sorry for being pedantic, but i think it goes a long way for the overall quality of the article.

File: src/problem-analysis/multiplayer.tex [hunk 3]
```diff
@@ -1,46 +1,41 @@
-\section{Multiplayer analysis}\label{sec:multiplayer-analysis}
+\section{Multiplayer}\label{sec:multiplayer-analysis}
 
 Chess is famously played by two players, where each player takes turns to move their pieces.
 Because of this, the game is inherently a multiplayer game.
 There are however different types of multiplayer games, and while chess is originally a tabletop game, adapting it to a
 digital platform has an effect on the multiplayer aspect of the game.
+This section will go over the different types of multiplayer games, and how the transition from local to online
+multiplayer has an effect on learning chess.
```
Comment (MadsHeilmann) [inline src/problem-analysis/multiplayer.tex]: Maybe mention that it has a "positive" effect to be more clear

File: src/problem-analysis/multiplayer.tex [hunk 4]
```diff
@@ -1,46 +1,41 @@
...
 
-From here on, this report will refer to the first type as local multiplayer and the third type as online
+\end{itemize}
+
+It can be concluded that by adapting chess to a digital platform, its multiplayer changes from local to online
 multiplayer.
-As previously discussed, chess as a tabletop is categorized as a local multiplayer game, while chess as a digital game
-is categorized as an online multiplayer game.
+While the game itself remains the same, the way players interact with each other changes drastically.
+This can have a negative effect of the experience players have when learning and playing chess.
```
Comment (MadsHeilmann) [inline src/problem-analysis/multiplayer.tex]: Is is not the opposite we are arguing? This sentence reads as if going from local to online is bad for learning. That may be true, but maybe then include a new sentence below saying something like "However, in the following section the benefits....". Right now it is like a "bad" cliffhanger before the next section which argues the opposite.

**Review by MadsHeilmann — CHANGES_REQUESTED** [review]
I think the pictures included in the report should be in light mode, as that just looks nicer on the paper IMO. 

I also think that the front-end section reads to much like "redegørelse" or like a guide. Maybe i am wrong, but should it not have more analysis of why different design choises were made and how they enable learning and so on?

It is also to be considered to only include pictures, wherein there is no reference to the emulator, as that is not part of the product - only the development environment. Maybe we can just create an issue for that to remember to edit it later on.

File: src/problem-analysis/multiplayer.tex (hunk 4 above)
Comment (Danar435) [reply src/problem-analysis/multiplayer.tex]: You're right. The next section explains how online games have both upsides and downsides, so I think that a neutral stance is more correct.

File: src/problem-analysis/multiplayer.tex (hunk 3 above)
Comment (Danar435) [reply src/problem-analysis/multiplayer.tex]: Refrence my other comment.

**Comment by SebastianMygind** [comment]
IMPORTANT:

I have messed with some commits so Mads got some of his commit deleted, so it would probably be best to wait with adding more until it is fixed. AKA please don't merge before we fix the conflicts/deletions.

**Comment by Danar435** [comment]
I've added arguments behind some of the design choices and switched the images to light mode. I'm not sure if they look better, because the background blends in with the paper, but I can revert back if you're against it.

**Review by audio-engineer — APPROVED** [review]

**Comment by MadsHeilmann** [comment]
You can maybe add a border around the images? I think that will look better.

**Review by SebastianMygind — DISMISSED** [review]

**Review by audio-engineer — DISMISSED** [review]

**Review by audio-engineer — APPROVED** [review]

**Review by MadsHeilmann — APPROVED** [review]

**Pull request merged**